tests/streamlit_ui_tests/graph.spec.ts

```diff
@@ -44,13 +44,41 @@ const testDeleteTitleThroughContextMenu = async (page, selector) => {
     await expect(mito.locator(selector)).not.toBeVisible();
 };
 
+const addColumnToAxis = async (mito: FrameLocator, page: Page, axis: 'X' | 'Y', columnName: string) => {
+  await mito.locator('.spacing-row', { hasText: `${axis} axis` }).locator('.mito-dropdown-button').click();
+  await mito.locator('.mito-dropdown-item', { hasText: columnName }).click();
+  await awaitResponse(page);
+}
+
+const openGraphEditor = async (mito: FrameLocator, page: Page) => {
+  await clickButtonAndAwaitResponse(page, mito, { name: 'Graph' })
+  await expect(mito.locator('.g-gtitle')).toBeVisible();
+}
+
+const changeChartType = async (mito: FrameLocator, page: Page, chartType: string, chartSubType?: string, nth?: number) => {
```

Comment:
It was confusing to me reading this utility when I would want to use `nth` / what it is. 

I think we use it if you are looking at a grouped bar chart to select between the `horizontal` and `vertical` ones. 

Maybe instead we could set the argument to `orientation: 'horizontal' | 'vertical' and then inside of the function we can convert those strings to 0 or 1 to be used in the nth function?

That would be easier for us to consume going forward probably.

Reply:
Ah yeah that's true, that's the only case that they would have the same `chartSubType` value.